tests/cli/run_analyzer/unit/test_writeconfig.py

```diff
@@ -17,10 +18,11 @@ def test_create_config():
             content = f.read()
 
         # Clean up
-        os.unlink(tmp.name)
+        # os.unlink(tmp.name)
 
         # Verify content
-        expected = """
+        expected = textwrap.dedent(
+            """process {
 withName: task1 {
     cpus = 2
     memory = 4GB
```

Comment:
The Nextflow docs suggests the pattern should be `4.GB`, this pattern is also used in NF-core workflow conf files? Do we need to change the generation logic to include the `.`?